channels/client/addin.h

```diff
@@ -16,3 +16,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* we need rdpContext */
+//#include <freerdp/freerdp.h>
+
+void* channel_client_create_handler(rdpContext* ctx, LPVOID userdata,
```

Comment:
these should most likely have `FREERDP_API` prefixed as channels might be compiled as separate libraries